### PR TITLE
#242 Interactive case study card

### LINKED ISSUE #242: Interactive case study card
### Description
Create the interactive case study card component as shown in the video attached below. 

### Animation Preview
- Screenshot/Video: 

https://github.com/user-attachments/assets/3f87d6f0-4a92-4011-8c23-86f89a3e06b2

> [!NOTE]
> Please refer to the **Animata contributing guidelines** available [here](https://www.animata.design/docs/contributing) for rules on how to contribute to the project. Let us know in the comments if you’re working on this issue, and feel free to ask any questions!

### Requirements
- [ ] Create a new animated component that matches the provided design.
- [ ] Add example usage of the component in the documentation or storybook.

<details>
  <summary>
    <h3> Guidelines & Best Practices</h3>
  </summary>

- Write reusable and clean code following the project’s style guide.
- The component should be **fully customizable**. The user should be able to modify props such as duration, easing, color, and size where applicable.
- Follow the [Contributing Guidelines](https://www.animata.design/docs/contributing) before submitting your pull request.

</details>

> [!IMPORTANT]  
> To ensure more contributors have the opportunity to participate, we kindly request that:
> - Each contributor submits only **one pull request** related to this issue.
> - If you're interested in working on this issue, first check if it has already been claimed. If not, **assign yourself** to the issue and move it to **In Progress** on the project board.

### Additional resources
Posted on X by: https://x.com/RomaTesla/status/1683429296939888640

---
### Checklist
- [ ] I have read and followed the contributing guidelines.
- [ ] I have developed the component according to the requirements.
- [ ] I have tested the component across different browsers and screen sizes.
- [ ] I have documented the component for future reference.
- [ ] I have submitted a pull request with appropriate labels and referenced this issue number.

### PR DESCRIPTION
**Pull request by webdevNYC**
Resolves #242 

Hey, Im creating the pull request now from the main.  

An interactive case study card that expands on hover, revealing a hidden tab with a clickable link to read comments.

### CODE REVIEW

**Comment by sudhashrestha** [comment]
@webdevNYC Please link this pull request to the respective issue. Check [this](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue).

File: content/docs/card/case-study-card.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Add the following to your tailwind.config.js file.
+
+```json
+module.exports = {
+  theme: {
+    extend: {
+    }
+  }
+}
+```
```
Comment (hari) [inline content/docs/card/case-study-card.mdx:29]: You don't need this step

File: animata/card/case-study-card.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      style={{
+        backgroundImage: `url(${image})`,
+        backgroundSize: "cover",
+        backgroundPosition: "center",
+      }}
+    />
+  );
+};
+
+export const CardAni = ({ show }: { show: React.ReactNode }) => {
```
Comment (hari) [inline animata/card/case-study-card.tsx:57]: Could you name it better? And I don't think you will need to export it.

File: content/docs/card/case-study-card.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+<ComponentPreview name="card-case-study-card--docs" />
+
+## Installation
+
+<Steps>
+<Step>Install dependencies</Step>
+
+```bash
+npm install framer-motion lucide-react
+```
```
Comment (hari) [inline content/docs/card/case-study-card.mdx:16]: You don't need this step as well

**Review by hari — CHANGES_REQUESTED** [review]

**Comment by hari** [comment]
Triggering workflow by closing and opening the PR.

**Comment by hari** [comment]
The build is failing as well @webdevNYC:

```
./animata/card/case-study-card.tsx:103:16
Type error: Property 'link' is missing in type '{ title: string | undefined; category: string | undefined; image: string; logo: string | undefined; }' but required in type 'CaseStudyCardProps'.

  101 |           show={
  102 |             type === "text"? (
> 103 |               <CardText title={title} category={category} image={image} logo={logo} />
      |                ^
  104 |             ) : (
  105 |               <CardImage image={image} title={title} />
  106 |             )
```

File: content/docs/card/case-study-card.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+title: Case Study Card
+description: An interactive case study card that expands on hover, revealing a hidden tab with a clickable link to read comments.
+author: @parankarj
```
Comment (hari) [inline content/docs/card/case-study-card.mdx:4]: Please remove `@`, it is causing the build failure.

Comment (hari) [reply content/docs/card/case-study-card.mdx:4]: @webdevNYC

File: content/docs/card/case-study-card.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+title: Case Study Card
+description: An interactive case study card that expands on hover, revealing a hidden tab with a clickable link to read comments.
+author: parankarj
+---
+
+<ComponentPreview name="card-case-study-card--docs" />
+
+## Installation
+
```
Comment (hari) [inline content/docs/card/case-study-card.mdx:10]: <Steps> is missing here. Please check locally once!

Comment (webdevNYC) [reply content/docs/card/case-study-card.mdx:10]: Im sorry, I was running only the story.

**Pull request merged**